### DISCUSSION TITLE
Listing relations in collection by collection name?

### DISCUSSION THREAD

**Discussion by baranelitez** [directus/directus#6031] [Archive]
Hi. As demonstrated in the docs [docs](https://docs.directus.io/reference/api/system/relations/#list-relations-in-collection) , I was trying to use the API to list relations under a specific collection, but it failed with a 403 error.

I'm using the injected `system.api` for my requests, but went ahead and made `Directus Relations` public, just for faster testing with my browser.

The thing that surprised me is that I can retrieve the relations of a collection if I put the collection ID number as a parameter, instead of the name. Though I couldn't find a way to get a collection's ID. So:
while `GET /relations/articles` does not work, `GET /relations/3` works.

Is that a bug? A documentational oversight? Or am I doing something very wrong? Thanks!

**Comment by rijkvanzanten** [comment]
@baranelitez Make sure your Directus installation is up to date, the structure for relations was changed very recently, so you might be looking at docs newer than the version you're currently using 🙂

**Reply by baranelitez** [reply]
yes, I can see that the relations docs were updated 15 days ago but I just started using directus! (probably ~48 hours ago). So I'm pretty sure I'm on the latest version.

**Reply by rijkvanzanten** [reply]
> So I'm pretty sure I'm on the latest version.

I'm not sure sure about that 😁  This:

> Though I couldn't find a way to get a collection's ID. So: while `GET /relations/articles` does not work, `GET /relations/3` works.

sounds to me like you're on <= rc.69 instead of >= rc.70

**Reply by baranelitez** [reply]
If you say so, I probably am hahah 😄 . I'll check tomorrow morning when I get back to working on it again, and will let you know!

**Reply by baranelitez** [reply]
It was indeed rc.69! Thanks!
